exporter/googlemanagedprometheusexporter/factory.go

```diff
@@ -37,10 +37,10 @@ const (
 )
 
 // NewFactory creates a factory for the googlecloud exporter
-func NewFactory() exporter.Factory {
+func NewFactory(collectorVersion string) exporter.Factory {
```

Comment:
Figured I'd mention that this breaks compatability with the opentelemetry builder tool, https://github.com/open-telemetry/opentelemetry-collector/tree/main/cmd/builder

Reply:
Spoke offline and we're ok with this risk as these changes are on wrappers of existing OTel exporters. Users will likely use the OTel version of those exporters rather than ours.